lib/show_task.rb

```diff
@@ -28,12 +28,25 @@ def show(description, scopes, filter, options = {})
 
   private
 
-  def show_html(description)
+  def show_html(description, options)
     begin
       LocalSystem.validate_existence_of_package("xdg-utils")
-      Html.generate(description)
-      html_path = SystemDescriptionStore.new.html_path(description.name)
-      LoggedCheetah.run("xdg-open", html_path)
+
+      url = "http://#{options[:ip]}:#{options[:port]}/#{CGI.escape(description.name)}"
+
+      Machinery::Ui.use_pager = false
+      Machinery::Ui.puts <<EOF
+There is a webserver running, serving the description on #{url}.
```

Comment:
Can you remove the period (`.`) to ease copy&paste (similar as in: https://github.com/SUSE/machinery/pull/1030/files#diff-92c36702dc702d09e7599c431e9ad5efR24 )?